src/domain/occurrence/constants.ts

```diff
@@ -0,0 +1,8 @@
+export enum EVENT_LANGUAGES {
```

Comment:
Maybe we should not duplicate these languages. Can be found already on /src/domain/event/constants.ts